### DISCUSSION TITLE
how to move the position of the RichTextEditor's toolbar in certain situations where it overlaps the top of the text box component

### DISCUSSION THREAD

**Discussion by mattnoth** [GrapesJS/grapesjs#3789] [Q&A] [answered]
![overlapping RTE](https://user-images.githubusercontent.com/6341816/133853696-07fff8e3-9d74-4bf1-aa4c-7815c0739276.png)

**Comment by artf** [comment, marked as answer]
Your question is already [here](https://github.com/artf/grapesjs/discussions/3782#discussioncomment-1349306). Don't create duplicate posts, please...
